cli/src/program_v4.rs

```diff
@@ -28,11 +28,14 @@ use {
     solana_compute_budget::compute_budget::ComputeBudget,
     solana_feature_set::{FeatureSet, FEATURE_NAMES},
     solana_instruction::Instruction,
-    solana_message::Message,
-    solana_program::loader_v4::{
-        self, LoaderV4State,
-        LoaderV4Status::{self, Retracted},
+    solana_loader_v4_interface::{
```

Comment:
This change from `solana_program::loader_v4` to `solana_loader_v4_interface` is a bit confusing. Why haven't you used the interface from the beginning? Or have you introduced it only recently?

Reply:
Yes, it was changed recently (https://github.com/anza-xyz/agave/pull/4486) as the SDK is being split into many individual crates.